test/Npgsql.Tests/YBPoolingWrapperTests.cs

```diff
@@ -0,0 +1,235 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace YBNpgsql.Tests;
+
+public class YBPoolingWrapperTests : YBTestUtils
+{
+    [Test]
+    public void TestPoolingForMultileConnStrings()
+    {
+        var connStringBuilder1 = "host=127.0.0.1;database=yugabyte;userid=postgres;password=postgres;Load Balance Hosts=any;Timeout=0;";
+        var connStringBuilder2 = "host=127.0.0.1;database=yugabyte;userid=tester;password=abc123;Load Balance Hosts=any;Timeout=0;";
+
+
+        List<NpgsqlConnection> conns = new List<NpgsqlConnection>();
+        CreateCluster();
+
+        try
+        {
+            NpgsqlConnection conn = new NpgsqlConnection(connStringBuilder1);
+            NpgsqlConnection conn1 = new NpgsqlConnection(connStringBuilder1);
+            NpgsqlConnection conn2 = new NpgsqlConnection(connStringBuilder2);
+            conn.Open();
+            conn.Close();
+            conn1.Open();
+            for (var i = 0; i < 6; i++)
+            {
+                NpgsqlConnection conn3 = new NpgsqlConnection(connStringBuilder1);
+                conn3.Open();
+                conns.Add(conn3);
```

Comment:
`conns` not being used anywhere. Did you intend to use this to close connections?